richtextfx-demos/src/main/java/org/fxmisc/richtext/demo/hyperlink/TextHyperlinkArea.java

```diff
@@ -0,0 +1,69 @@
+package org.fxmisc.richtext.demo.hyperlink;
+
+import javafx.geometry.VPos;
+import org.fxmisc.richtext.GenericStyledArea;
+import org.fxmisc.richtext.TextExt;
+import org.fxmisc.richtext.model.ReadOnlyStyledDocument;
+import org.fxmisc.richtext.model.StyledText;
+import org.fxmisc.richtext.model.TextOps;
+import org.reactfx.util.Either;
+
+import java.util.function.Consumer;
+
+public class TextHyperlinkArea extends GenericStyledArea<ParStyle, Either<StyledText<TextStyle>, Hyperlink<TextStyle>>, TextStyle> {
```

Comment:
This demo seems like a lot of code just to demo hyperlinks. And it isn't clear to me at a quick glance if something like ParStyle is part of the minimal code necessary to create a hyperlink or if it's part of the bells and whistles of the demo.